docs/rules/prefer-user-event.md

```diff
@@ -0,0 +1,126 @@
+# Use [userEvent](https://github.com/testing-library/user-event) over using `fireEvent` for user interactions (prefer-user-event)
+
+From
+[testing-library/dom-testing-library#107](https://github.com/testing-library/dom-testing-library/issues/107):
+
+> [...] it is becoming apparent the need to express user actions on a web page
+> using a higher-level abstraction than `fireEvent`
+
+`userEvent` adds related event calls from browsers to make tests more realistic than its counterpart `fireEvent`, which is a low-level api.  
+See the appendix at the end to check how are the events from `fireEvent` mapped to `userEvent`
+
+## Rule Details
+
+This rules enforces the usage of [userEvent](https://github.com/testing-library/user-event) methods over `fireEvent`. By default, the methods from `userEvent` takes precedence, but you add exceptions by configuring the rule in `.eslintrc`
+
+See below for examples of valid usages of `fireEvent` methods with the configuration.
```

Comment:
Personally, I think this line can be removed - or that it can be replaces with a link to the options.
I'm fine either way tho, this was the only comment I would not have said it.
```suggestion
```